src/super_gradients/sanity_check/env_sanity_check.py

```diff
@@ -0,0 +1,165 @@
+            "==": installed_version == required_version
+        }
+        if not is_constraint_respected[constraint]:
+            errors.append(
+                f"{lib} is installed with version {installed_version} which does not satisfy {requirement} (based on {requirements_path})")
+    return errors
+
+
+def verify_os() -> List[str]:
+    """Verifying operating system name and platform"""
+    name = os.name
+    platform = sys.platform
+    logger.info(f'OS Name: {name}')
+    logger.info(f'OS Platform: {platform}')
+    if 'linux' not in platform.lower():
+        return ['Deci officially supports only Linux kernels. Some features may not work as expected.']
+    return []
+
+
+def env_sanity_check():
+    """Run the sanity check tests and log everything that does not meet requirements"""
+
+    keep_log_level = logger.level
+
+    # By default we don't deplay sanity logs, unless DISPLAY_SANITY_CHECK_VAR=True
+    display_sanity_check = (os.getenv(DISPLAY_SANITY_CHECK_VAR, "False") == "True")
+    logger.setLevel(logging.INFO if display_sanity_check else logging.WARNING)
+
+    requirement_checkers = {
+        'operating_system': verify_os,
+        'libraries': verify_installed_libraries,
+    }
+
+    logger.info('SuperGradients Sanity Check Started')
+    logger.info(f'Checking the following components: {list(requirement_checkers.keys())}')
+    logger.info('_' * 20)
+
+    lib_check_is_impossible = False
+    sanity_check_errors = {}
+    for test_name, test_function in requirement_checkers.items():
+        logger.info(f"Verifying {test_name}...")
+
+        errors = test_function()
+        if errors == [LIB_CHECK_IMPOSSIBLE_MSG]:
+            lib_check_is_impossible = True
+            logger.info(LIB_CHECK_IMPOSSIBLE_MSG)
+        elif len(errors) > 0:
+            sanity_check_errors[test_name] = errors
+            for error in errors:
+                logger.info(f"Failed to verify {test_name}: {error}")
+        else:
+            logger.info(f'{test_name} OK')
+        logger.info('_' * 20)
+
+    if sanity_check_errors:
+        logger.info(
+            f'The current environment does not meet Deci\'s needs, errors found in: {", ".join(list(sanity_check_errors.keys()))}')
+    elif lib_check_is_impossible:
+        logger.info(LIB_CHECK_IMPOSSIBLE_MSG)
+    else:
+        logger.info('Great, Looks like the current environment meet\'s Deci\'s requirements!')
+
+    # The last message needs to be displayed independently of DISPLAY_SANITY_CHECK_VAR
+    logger.setLevel(keep_log_level)
+    if display_sanity_check:
+        logger.info(f'**This check can be hidden by setting the env variable {DISPLAY_SANITY_CHECK_VAR}=False prior to import**')
+    else:
+        logger.info(f'**A sanity check is done when importing super_gradients for the first time.'
+                    f'You can see the details by setting the env variable {DISPLAY_SANITY_CHECK_VAR}=True prior to import**')
```

Comment:
you mean =False

Reply:
I changed variable names, now it should be good